docs/source/_templates/sphinx_conf.template.py

```diff
@@ -34,12 +34,7 @@
 extensions = [
     "sphinx_design",
     "sphinx.ext.autodoc",
-    "sphinx.ext.mathjax",
-    "sphinx.ext.napoleon",
-    "sphinx.ext.viewcode",
-    "sphinxarg.ext",
```

Comment:
This extension is installed by brassy, so does not increase our installation size by including it here. The hope is that the GeoIPS CLI will use this extension at some point to document the CLI